modules/enterprise/server/server-metrics/src/main/java/org/rhq/server/metrics/aggregation/DataAggregator.java

```diff
@@ -351,8 +351,10 @@ public AggregationTask(Batch batch) {
         @Override
         public void run() {
             try {
+                permits.acquire();
                 run(batch);
             } catch (Exception e) {
+                permits.release();
                 log.error("Aggregation will be aborted due to an unexpected error", e);
                 taskTracker.abort("Aborting aggregation due to an unexpected error: " + e.getMessage());
```

Comment:
Yea, it's not. I'm assuming it's still running the CQL command and that's just "stuck" or something along those lines.